tests/trainer/properties/log_dir.py

```diff
@@ -140,3 +140,18 @@ def test_logdir_custom_logger(tmpdir):
     assert trainer.log_dir == expected
     trainer.fit(model)
     assert trainer.log_dir == expected
+
+
+def test_logdir_logger_collection(tmpdir):
+    """Tests that the logdir equals the default_root_dir when the logger is a LoggerCollection"""
+    expected = tmpdir
+    model = TestModel(expected)
+
+    trainer = Trainer(
+        default_root_dir=tmpdir, max_steps=2, logger=[TensorBoardLogger(save_dir=tmpdir, name='custom_logs')]
```

Comment:
if you are testing that the expected log dir should be the default_root_dir, you should set the same value for save_dir in the tensorboard logger. choose a different path so the test can never give a false positive. 
hope it's clear what I mean :)

Reply:
@gahdritz Could you address @awaelchli's comment?

Reply:
@awaelchli can you confirm https://github.com/PyTorchLightning/pytorch-lightning/pull/8187/commits/6941d930a4edfd94d960132bc487025e025aabec is what you meant?

Reply:
yes, thanks
that's what I meant